app/src/main/java/com/github/se/cyrcle/ui/authentication/SignInScreen.kt

```diff
@@ -85,58 +94,90 @@ fun SignInScreen(navigationActions: NavigationActions, userViewModel: UserViewMo
                     when (it) {
                       UserViewModel.SignInFailureReason.ACCOUNT_NOT_FOUND -> {
                         Toast.makeText(context, accountNotFoundToast, Toast.LENGTH_LONG).show()
+                        navigationActions.navigateTo(Screen.CREATE_PROFILE)
```

Comment:
As Louis said it is a clean way of refactoring the create account button into the google Sign in one, good and clean job !